os/drivers/wireless/scsc/slsi_drv_netmgr.c

```diff
@@ -599,7 +599,7 @@ trwifi_result_e slsidrv_connect_ap(struct netdev *dev, trwifi_ap_config_s *ap_co
 	} else {
 		result = TRWIFI_SUCCESS;
 		vdvdbg("Successfully joined the network: %s(%d)\n", ap_connect_config->ssid,
-			  ap_connect_config->ssid_length);
+			   ap_connect_config->ssid_length);
```

Comment:
Please check the indentation. There are extra spaces before the line.

Reply:
I think it's alright,
It's not a new line of code but a continued one, hence the extra 2 spaces.